dags/incline_dash.py

```diff
@@ -59,49 +58,62 @@
         email=["telemetry-alerts@mozilla.com", "frank@mozilla.com"],
         dag=dag)
 
-    sql = (
-        'SELECT * FROM `{project}.{dataset}.{table}` WHERE date = "{date}"'
-    ).format(project=project, dataset=dataset, table=table, date="{{ ds }}")
-
-    gcp_conn_id = 'google_cloud_derived_datasets'
-    fully_qualified_tmp_table = (
-        "{project}.tmp.{table}_{date}"
-        .format(project=project, table=table, date="{{ ds_nodash }}")
-    )
-
-    create_table = BigQueryOperator(
-        task_id='create_temporary_table',
-        sql=sql,
-        destination_dataset_table=fully_qualified_tmp_table,
-        bigquery_conn_id=gcp_conn_id,
-        use_legacy_sql=False
-    )
-
-    fully_qualified_table_name = '{}.{}.{}'.format(project, dataset, table)
-    gcs_bucket = 'moz-fx-data-prod-analysis'
-    incline_prefix = 'incline/executive_dash/{date}/data.ndjson'.format(date="{{ ds }}")
-    gcs_uri = 'gs://{bucket}/{prefix}'.format(bucket=gcs_bucket, prefix=incline_prefix)
-
-    table_extract = BigQueryToCloudStorageOperator(
-        task_id='extract_as_latest',
-        source_project_dataset_table=fully_qualified_table_name,
-        destination_cloud_storage_uris=[gcs_uri],
-        bigquery_conn_id=gcp_conn_id,
-        export_format='JSON',
-        compression='GZIP'
-    )
-
-    # Drop the temporary table
-    table_drop = BigQueryOperator(
-        task_id='drop_temp_table',
-        sql='DROP TABLE `{}`'.format(fully_qualified_tmp_table),
-        bigquery_conn_id=gcp_conn_id,
-        use_legacy_sql=False
-    )
-
     wait_for_copy_deduplicate >> \
         migrated_clients >> \
+        exec_dash
+
+    countries = ('US', 'CA', 'DE', 'FR', 'GB', 'IN', 'CN', 'IR', 'BR', 'IE', 'ID', 'tier-1', 'non-tier-1')
+    for country in countries:
+        bq_country = country.replace('-', '_')
+
+        sql = (
+            'SELECT * FROM `{project}.{dataset}.{table}` WHERE country = {country}'
+        ).format(project=project, dataset=dataset, table=table, country=country)
+
+        fully_qualified_tmp_table = (
+            "{project}.tmp.{table}_{country}_{date}"
+            .format(project=project, table=table, country=bq_country, date="{{ ds_nodash }}")
+        )
+
+        gcp_conn_id = 'google_cloud_derived_datasets'
+        create_table = BigQueryOperator(
+            task_id='create_temporary_table',
+            sql=sql,
+            destination_dataset_table=fully_qualified_tmp_table,
+            bigquery_conn_id=gcp_conn_id,
+            use_legacy_sql=False
+        )
+
+        gcs_bucket = 'moz-fx-data-prod-analysis'
+        incline_prefix = 'incline/executive_dash/{}/' + '{}.csv.gz'.format(country)
```

Comment:
```suggestion
        incline_prefix = 'incline/executive_dash/{}/' + '{}.csv.gz'.format(country)
```

Can we call this `incline_prefix_template` to make it more obvious that a `.format` is going to be called on it later? I also think we should make it a named placeholder rather than positional. Maybe call it `date`?